src/sagemaker_containers/environment.py

```diff
@@ -51,6 +55,11 @@
 RESOURCE_CONFIG_FILE = 'resourceconfig.json'  # type: str
 INPUT_DATA_CONFIG_FILE = 'inputdataconfig.json'  # type: str
 
+HYPERPARAMETERS_PATH = os.path.join(INPUT_CONFIG_PATH, HYPERPARAMETERS_FILE)  # type: str
+INPUT_DATA_CONFIG_FILE_PATH = os.path.join(INPUT_CONFIG_PATH, INPUT_DATA_CONFIG_FILE)  # type: str
+RESOURCE_CONFIG_PATH = os.path.join(INPUT_CONFIG_PATH, RESOURCE_CONFIG_FILE)  # type: str
+
+PROGRAM_PARAM = 'sagemaker_program'  # type: str
```

Comment:
same as above, USER_PROGRAM_PARAM is defined just below this line.

Reply:
I will fix it.

Reply:
You missed this one.

Reply:
I missed because they don't have the same name. Sorry about that.